demo.ipynb

```diff
@@ -158,6 +158,34 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "Next, let's compare the quantile approximation to the truth using the Kullback-Leibler DIvergence."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "grid1 = np.linspace(-1., 1., 200)\n",
+    "grid2 = np.linspace(-2., 2., 400)\n",
+    "grid3 = np.linspace(-3., 3., 600)\n",
+    "\n",
+    "D1 = P.kld(grid1)\n",
+    "D2 = P.kld(grid2)\n",
+    "D3 = P.kld(grid3)\n",
+    "\n",
+    "print(D1, D2, D3)"
+   ]
+  },
+  {
    "cell_type": "code",
```

Comment:
Can you provide some commentary on the results of the KLD calculation here? Do the values make sense? Do we want the KLD to be close to zero or what? How does the magnitude of the KLD (in bits) relate to sentences we would like to write, which are of the form "the accuracy of the approximation is X"? I guess you might want to implement some other simple metrics as well, such as root mean square error, suitably (ie standardly!) normalized.